open_dread_rando/door_patcher.py

```diff
@@ -149,14 +151,22 @@ def is_door(actor: Container):
 class DoorPatcher:
     """An API to patch doors. Call patch_door() to use. """
 
-    def __init__(self, editor: PatcherEditor, show_shields: bool = False) -> None:
+    def __init__(self, editor: PatcherEditor) -> None:
         # get actors from reference dicts
         self.editor = editor
-        self.show_shields_on_minimap = show_shields
+        self.available_shield_ids = {
+            "s010_cave": list(range(0,100)),
```

Comment:
`range` by default starts at 0.

Also, this could be simplified to this:
```python
self.available_shield_ids = {
    scenario: list(range(100))
    for scenario in ALL_SCENARIOS
}
```